config/schedule.rb

```diff
@@ -0,0 +1,31 @@
+set :environment, 'development'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.